src/Functions/getMaxTableNameLength.cpp

```diff
@@ -92,7 +92,37 @@ class FunctionGetMaxTableNameLength : public IFunction, WithContext
 
 REGISTER_FUNCTION(getMaxTableName)
 {
-    factory.registerFunction<FunctionGetMaxTableNameLength>();
+    factory.registerFunction<FunctionGetMaxTableNameLength>(FunctionDocumentation{
```

Comment:
Side note: Suggest to also rename the class name `FunctionGetMaxTableNameLength` to `FunctionGetMaxTableNameLengthForDatabase`.

Reply:
done